src/commands/package.ts

```diff
@@ -36,21 +39,37 @@ export default class Package extends Command {
             return;
         }
 
-        const compiler = new AppCompiler(this, fd);
-        const report = compiler.logDiagnostics();
+        const compiler = new AppsCompiler({
+            tool: packageInfo.name,
+            version: packageInfo.version,
+            when: new Date(),
+        }, this.getTypescriptForApp(fd) as any); // typing this is hard
 
-        if (!report.isValid && !flags.force) {
-            this.error('TypeScript compiler error(s) occurred');
+        const result = await compiler.compile(fd.folder);
+
+        if (result.diagnostics.length) {
```

Comment:
We should still keep the `!flags.force` option for when someone needs to package an invalid one.